headerItems.js

```diff
@@ -1,34 +1,32 @@
 const request = require('request');
-var fs = require('fs');
-var output = "";
 
-request('https://refdata-api.azurewebsites.net/api/fullheadermenu', function (error, response, body) {
-  if (!error && response.statusCode == 200) {
-    var topLevel = JSON.parse(body);
-    for(var i = 0; i < topLevel.length; i++) {
-      var item = topLevel[i];
-      var itemURL = item.URL.split(",")
-          itemURL = itemURL[0].replace('site','www.nhs.uk');
-      var subItems = item.Submenus;
-      output += '<li class=""><a href="'+itemURL+'">'+item.Title+'</a>';
-      if (subItems.length != 0) {
+const fs = require('fs');
+
+let output = '';
+
+request('https://refdata-api.azurewebsites.net/api/fullheadermenu', (error, response, body) => {
+  const topLevel = JSON.parse(body);
+  if (!error && response.statusCode === 200) {
+    for (let i = 0; i < topLevel.length; i++) {
+      const item = topLevel[i];
+      let itemURL = item.URL.split(',');
+      itemURL = itemURL[0].replace('site', 'www.nhs.uk');
```

Comment:
To be a bit safer I'd suggest replacing a bit more than just `site` and it might as well go to `https`.
How about this?
```js
itemURL = itemURL[0].replace('http://site', 'https://www.nhs.uk');
```
Also, one of the URLs (http://www.nhs.uk/Service-Search/Care at home/LocationSearch/1833) has a space in it that could do with replacing with `%20`.

Reply:
Done